tests/ci-helper.test.ts

```diff
@@ -304,6 +311,7 @@ testQ("handle comment allow fail invalid user", async () => {
     };
 
     ci.setGHGetPRComment(comment);
+    ci.letGHGetGitHubUserInfoThrow("is not a valid GitHub username");
```

Comment:
This test was originally intended to verify this line was run in `CIHelper`:
```
                    throw new Error(`User ${accountName} is not a valid GitHub username: ${reason}`);
```
Running the tests locally, the error is `Error: User bad_@@@@ is not a valid GitHub username: Error: Need a GitHub token for gitgitgadget`, which has as the reason an error thrown by `GitHubGlue` (and no network access).  If you want to verify there is no network while still allowing the original test, how about:
```
   const noNetwork = "No net";
   ci.letGHGetGitHubUserInfoThrow(noNetwork);
   await ...
   expect ...
   expect(ci.addPRCommentCalls[0][1]).toMatch(noNetwork); // new test for no network access
```

Reply:
Oh wow, never mind! I must have been in a situation when I developed the patch originally where I had lost network access (being happy about Git's distributed nature!). However, I just reran the test suite after disconnecting from the network, and it all passed:

```
Test Suites: 26 passed, 26 total
Tests:       152 passed, 152 total
Snapshots:   0 total
Time:        115.744 s, estimated 116 s
Ran all test suites.
```

I should have verified whether this patch is still needed before opening this PR, but I [keep forgetting that when rebasing branch thickets](https://github.com/gitgitgadget/gitgitgadget/pull/1473#discussion_r2009052927), sorry!!!